packages/hawtio/src/plugins/camel/CamelContent.tsx

```diff
@@ -59,6 +59,8 @@ export const CamelContent: React.FunctionComponent = () => {
   }
 
```

Comment:
Might just want a comment here documenting that this array is ordered in a particular way and document that order as a list. Otherwise, people like me will just come along and add another view in the middle without thinking about it :-)

Reply:
Good idea, I'll add it

Reply:
Er forgot about this and I have to go. I'll add it tomorrow

Reply:
I did it up from GitHub UI